docs/internals/deprecation.txt

```diff
@@ -125,6 +125,8 @@ details on these changes.
 
 * The ``--list`` option of the ``migrate`` management command will be removed.
 
+* The :ttag:`ssi` template tag will be removed.
```

Comment:
Removing the link syntax prevents updates in upcoming releases when the reference isn't available anymore.